polkadot/node/core/pvf/prepare-worker/src/lib.rs

```diff
@@ -20,6 +20,7 @@ mod executor_intf;
 mod memory_stats;
 
 pub use executor_intf::{prepare, prevalidate};
+use libc;
```

Comment:
Ideally we don't mix `libc` and whatever safe library we use. For example, if we use `rustix`, it exposes `rustix::process::EXIT_SUCCESS`.